ext/gd/tests/bug77269.phpt

```diff
@@ -4,6 +4,7 @@ Bug #77269 (Potential unsigned underflow in gdImageScale)
 <?php
 if (!extension_loaded('gd')) die('skip gd extension not available');
 if (getenv("SKIP_SLOW_TESTS")) die("skip slow test");
+if (substr(PHP_OS, 0, 3) == 'WIN' && PHP_INT_SIZE === 4) die("skip Windows 64-bit only"); ?>
```

Comment:
why is the borking the test https://github.com/php/php-src/actions/runs/3250527885/jobs/5334323268#step:10:1493 for non-Windows?